docs/source/development/customizing-components.md

```diff
@@ -1,100 +1,90 @@
+## Customizing the `Logo` resource
 
-So, for example, if we want to replace the Logo which is located in
-Volto at `components/theme/Logo/Logo.svg`, the folder structure needs
-to match the folder structure of Volto in the `customizations` folder.
-So the final path of the new overridden component will
-be: `customizations/components/theme/Logo/Logo.svg`.
+So, for example, if you want to replace the logo which is located in Volto at {file}`components/theme/Logo/Logo.svg`, then the folder structure needs to match the folder structure of Volto in the {file}`customizations` folder.
+The final path of the new overridden component will be {file}`<my-volto-addon-or-package/>src/customizations/components/theme/Logo/Logo.svg`.
 
-## Change The Tags Component
+## Change the `Tags` component
 
-When overriding components, we follow the same approach. We will
-copy over the original component from the Volto source code, then amend the
-imports (if any are required) to match the current folder structure. Point Volto
-source code using `@plone/volto` module instead of relative paths and other
-amendments required.
+When overriding components, we follow the same approach.
+Copy over the original component from the Volto source code, then amend the imports (if any are required) to match the current folder structure.
+Point Volto source code to use the `@plone/volto` module, instead of relative paths, and make other required amendments.
 
 Locate the `Tags.jsx` file and override this file so that there is a label in front of the tags with: `Tags:`.
 
 ```{code-block} jsx
-:emphasize-lines: 20
+:emphasize-lines: 26
 /**
  * Tags component.
  * @module components/theme/Tags/Tags
  */
 
-import React from 'react';
-import { UniversalLink } from '@plone/volto/components';
+import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { Container } from 'semantic-ui-react';
+import config from '@plone/registry';
 
 /**
- * Tags component class.
+ * Tags component.
  * @function Tags
- * @param {array} tags Array of tags.
- * @returns {string} Markup of the component.
+ * @param {Object} props Component properties.
+ * @param {Object} props.content Content object that may contain subjects.
+ * @param {Array} [props.content.subjects] Optional array of tags (subjects).
+ * @returns {JSX.Element|null} Markup of the component or null if no tags are available.
  */
-const Tags = ({ tags }) =>
-  tags && tags.length > 0 ? (
-    <Container>
+const Tags = ({ content }) => {
+  const tags = content?.subjects || [];
+
+  if (!config.settings.showTags || !tags.length) return null;
+
+  return (
+    <Container className="tags">
       Tags:
       {tags.map((tag) => (
-        <UniversalLink
-          className="ui label"
-          href={`/search?Subject=${tag}`}
-          key={tag}
-        >
+        <Link className="ui label" to={`/search?Subject=${tag}`} key={tag}>
           {tag}
-        </UniversalLink>
+        </Link>
```

Comment:
What's wrong with the `UniversalLink`?

Reply:
Reverted in 0ef2ad02a0e6a3ad3fc2432538a1a326bfadc325.